ivy_tests/test_ivy/test_frontends/test_tensorflow/test_random.py

```diff
@@ -118,3 +118,51 @@ def test_tensorflow_shuffle(
         value=values[0],
         seed=seed,
     )
+
+
+#random_stateless_uniform
+@handle_frontend_test(
+    fn_tree="tensorflow.random.stateless_uniform",
+    shape=helpers.dtype_and_values(
+        available_dtypes=tuple([ivy.int64, ivy.int32]),
+        min_value=1,
+        max_value=5,
+        min_num_dims=1,
+        max_num_dims=1,
+    ),
+    minval=helpers.ints(min_value=0, max_value=3),
+    maxval=helpers.ints(min_value=4, max_value=10),
+    dtype=helpers.get_dtypes("float", full=False),
+    seed=helpers.ints(min_value=0, max_value=10),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_stateless_uniform(
+    shape,
+    seed,
+    minval,
+    maxval,
+    dtype,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+):
+    seed = ivy.array([seed,seed], dtype = "int32").data
```

Comment:
By adhering to the comment on generating seeds you will remove this line completely, as well as the `import ivy` line